pkgs/leak_tracker_flutter_testing/test/tests/matchers_test.dart

```diff
@@ -21,17 +21,21 @@ class _TrackedClass {
 }
 
 void main() {
-  test('dispatchesMemoryEvents success', () {
-    expect(
-      () => _TrackedClass().dispose(),
-      dispatchesMemoryEvents(_TrackedClass),
+  test('dispatchesMemoryEvents success', () async {
+    await expectLater(
```

Comment:
I don't think you need to await `memoryEvents` below as `expectLater` will do that.

Reply:
If I do not await, the matcher gets Future<List<ObjectEvent>>.